packages/backend/src/indexers/DiscoveryIndexer.ts

```diff
@@ -35,6 +35,17 @@ export class DiscoveryIndexer extends ChildIndexer {
     ])
   }
 
+  override async start(): Promise<void> {
+    const oldConfigHash = (
+      await this.indexerStateRepository.findById(this.id, this.chainId)
+    )?.configHash
+    const newConfigHash = this.config.hash
+    if (oldConfigHash !== newConfigHash) {
+      await this.setSafeHeight(0)
+    }
+    await super.start()
+  }
+
```

Comment:
Add some spacing, let the air in please

Reply:
[0c94dec](https://github.com/l2beat/lz-monitoring/pull/55/commits/0c94dec2412fa9e1b83b570991bd3104c64a4761)